robot_urdf_path: 'robots/allegro_hand/allegro_hand_description_left.urdf'

joint_upper_bound: [0.47, 1.61, 1.709, 1.618,0.47, 1.61, 1.709, 1.618,  0.47, 1.61, 1.709, 1.618, 1.396, 2, 1.644,  1.719]
joint_lower_bound: [-0.47, -0.196, -0.174, -0.227,  -0.47, -0.196, -0.174, -0.227, -0.47,  -0.196, -0.174, -0.227,0.263, -0.105, -0.189, -0.162]

JOINT_NAMES: [ 'joint_0.0', 'joint_1.0', 'joint_2.0', 'joint_3.0', 'joint_3.0_tip', 'joint_4.0', 'joint_5.0', 'joint_6.0', 'joint_7.0', 'joint_7.0_tip', 'joint_8.0', 'joint_9.0', 'joint_10.0', 'joint_11.0', 'joint_11.0_tip', 'joint_12.0', 'joint_13.0', 'joint_14.0', 'joint_15.0', 'joint_15.0_tip']